 
**About me** 
- Open to position on **Plant, Bacterial, Fungal Bioinformatics, Deep Learning including language models,and DevOPs**.
- If i have applied for your open position then i am surely interested.
- If i have previously applied to your position, and not been selected and still open then i am interested.
- I continously learn about the priority languages and areas of interest as in bioinformatics is null without informatics.
- Prior to 2022, all the code I wrote was internally hosted by the research institutions/universities.
- All code and ideas concieved and written by me unless specified and i am open to continous development.
- [**Must Read: One page description on my approach, professionalism, work attitude**](https://github.com/gauravcodepro/gauravcodepro/blob/main/work.md)
- Application package: [**Curriculum Vitae**](https://github.com/gauravcodepro/gauravcodepro/blob/main/Curriculum_Vitae_Gaurav_Sablok_2024.pdf), [**Scopus**](https://www.scopus.com/authid/detail.uri?authorId=36633064300), [**ORCID**](https://orcid.org/0000-0002-4157-9405), [**Web of Science**](https://www.webofscience.com/wos/author/record/C-5940-2014). [**Employers references**](https://github.com/gauravcodepro/gauravcodepro/blob/main/references.pdf).
- Approachable at [**gauravcodepro@gmail.com**](mailto:gauravcodepro@gmail.com)
- Voracious reader of [**language-models**](https://paperswithcode.com/)
  
**Areas of Interest**
- **Bioinformatics,Deep Learning,RAG,Language Models and Graphs.**
  - Priority Language: Python, R, Ruby, Julia, Javascript, Shell, Awk, Nextflow
  - Machine/Deep Learning:PyTorch, Tensorflow, CatBoost, XGBoost, Scikit-learn, Keras, LightGBM, Fastai.
  - Package Development: Python, Julia, Ruby.
  - API Development using FASTAPI
  - Documentation: Markdown, ReadTheDocs, MkDocs, Jekyll, Github Pages, Quatro
- **DevOPs**
  - Priority Language: Shell, NuShell, Awk, Python, Nodejs
  - Code Management: Git, GitKraken.
  - Application deployment: [Docker](https://hub.docker.com/u/gauravcodepro)
  - Orchestration:Terraform, Kubernetes.
  - Database Management: SQlite3, MySQL, MongoDB.
  - HPC management: PBS and SLURM as job scheduler
- **Web Develpoment**
  - Priority Language: Ruby on Rails, Streamlit, Python Shiny/ShinyExpress, Django, Julia, Flask, HTML, Javascript, Typescript
- **Application Development**
  - Priority Language: Electron, PyQT, PyGTK

- [**Python Packages**](https://pypi.org/user/gauravcodepro/) | [**Python Community**](https://www.python.org/community/)  ➡️  [**Tairaccession**](https://github.com/gauravcodepro/tairaccession) | [**Graphanalyzer**](https://github.com/gauravcodepro/graphanalyzer) | [**RNAprocessor**](https://github.com/gauravcodepro/rnaprocessor) | [**ProteinAnnotator**](https://github.com/gauravcodepro/protein-annotator) | [**Ontologyanalyzer**](https://github.com/gauravcodeproontologyanalyzer)
- [**Streamlit Applications**](https://streamlit.io/) | [**Streamlit Community**](https://discuss.streamlit.io/) ➡️ [**Metabolic-analyzer:streamlit API to BIGG database models**](https://github.com/gauravcodepro/BIGG-metabolic-analyzer-API) | [**Metabolic-json: streamlit analyzing BIGG metabolic models**](https://github.com/gauravcodepro/metabolic-json-modelling) | [**Ontology-graphs: streamlit analyzing arabidopsis ontologies and plant obo**](https://github.com/gauravcodepro/arabidopsis-ontology-graphs) | [**Pacbiohifi:streamlit pacbiohifi sequencing information**](https://github.com/gauravcodepro/pacbiohifi) | [**DevOPS streamlit application slurm generator**](https://github.com/gauravcodepro/universitat-potsdam-devops-application)
- [**Ruby gems**](https://rubygems.org/profiles/gauravcodepro) | [**Ruby app**](https://www.ruby-forum.com/) | [**Ruby Community**](https://www.ruby-forum.com/) ➡️ [**Devopsutils**](https://github.com/gauravcodepro/devops-system) | [**Ruby-template**](https://github.com/gauravcodepro/ruby_gem_creator) | [**Fasta-tokenizer**](https://github.com/gauravcodepro/pacbiohifi-motif-scanner) | [**Panacheextract**](https://rubygems.org/gems/panacheextract) | [**Protalign**](https://github.com/gauravcodepro/proteinalignment-annotation-gem) and [**proteinmultialign**](https://github.com/gauravcodepro/protein-multialign-gem) | [**Pacbiohifi-analyzer**](https://github.com/gauravcodepro/pacbiohifi-analyzer) 

- **Electron Application**➡️ [TAIR genome analysis](https://github.com/gauravcodepro/arabidopsis-tair-application)

**Work Attitude/Professionalism**
- I dont believe in those who say wait and see, as they are already showing that they are time wasters and today they are wasting yours, tomorrow someone else. Instead, i believe in those, grab the oppurtunity, work on the same and generate results as that will give prominecense and visibility.
- I dont dwelve into people personal choices such as browser,code editor, font, theme or operating system, or any other non-living thing as that is not important to me. No one cares about the code editor you used, they care that we have a working code. I dont spend time on whatever non-living thing you use as that has no meaning and after sometime, you throw them in trash or discard them so any decision made on such non-living momentary is a sign of mental illness. 
- I believe decisions and communication not made in time have no value and importance.
- I constantly read scientific aptitude/literature/code development to increase my work potential.
- Active in solving limitations, proficient in developing new approaches for code development.
- Extremely collaborative and open to supporting those who support me.
- I give importance and priority to the people I am working with.
- I fork repositories as per language and dependencies need and once learnt then unforked.
- Highly Communicative, approachable, motivated, task and solution oriented, time responsive, and initiative taker.
- Pro-active in making effective communication as communication leads to result oriented approaches.
