I read about what i code. My areas of focus are Bioinformatics, DataScience, Machine and Deep Learning. I program across Python, R, Shell, Nu, Hashicorp, Ruby, Awk, Crystal, R and Python shiny application and also use rust based nushell programming for many aspects from data exploration to bioinformatics and datascience. I work with large scale sequencing (transcriptomics, genome sequencing metagenomics and single cell genomics(plants specific) and also spatial transcriptomics) involving the assembly of the plant genomes from high order to lower order, chloroplast and mitochondrial genomes, bacterial genomes, metagenomics and fungal genomes coming from Illumina Seq, MiSeq, NextSeq, PacBio, Oxford Nanopore. Having an extensive experience in molecular biology, bioinformatics, comparative genomics and biological mining of large-scale data-sets and databases in plant, bacterial and fungal species to unravel the biological basis. Skilled in both pbs and slurm based high performance computing clusters, devops management and uses git, docker and kubernetes for container orchestration. If you would like to connect then you can connect with me below 

Github:https://github.com/sablokgaurav \
ORCID: https://orcid.org/0000-0002-4157-9405 \
WOS: https://www.webofscience.com/wos/author/record/C-5940-2014 \
RubyGems Published: https://rubygems.org/profiles/sablokgaurav \
Python Packages Published : https://pypi.org/user/sablokgaurav/ and https://discuss.python.org/u/sablokgaurav/summary \
Posit community: https://community.rstudio.com/u/sablokgaurav/summary \
Crystal Profile: https://forum.crystal-lang.org/u/sablokgaurav/ \
devProfile: https://app.daily.dev/sablokgaurav \
Linux community: https://linuxcommunity.io/u/sablokgaurav/summary and https://linuxcommunity.io \
assumptions lead to failures and efforts lead to success.


