#### About me <img width="30" height="30" src="https://img.icons8.com/dotty/100/parse-resumes.png" alt="parse-resumes"/>
 - I work as Academic Staff Member at Bioinformatics, Institute for Biochemistry and Biology at [Universtat Potsdam(UP)](https://www.uni-potsdam.de/de/) and [Group](https://www.uni-potsdam.de/en/ibb-bioinformatik/members/gaurav-sablok) :de:
 - I work at the interface of both academics as well as developing applications from client side rendering. 
 - You can contact me at [gaurav.sablok@uni-potsdam.de](mailto:gaurav.sablok@uni-potsdam.de)
 - If you want to contact me on Gmail, you can use [gauravcodepro@gmail.com](mailto:gauravcodepro@gmail.com)
 - Communicative, approachable, motivated, task and solution oriented, time responsive, and initiative taker. 
 - **Bioinformatics**: genomes, transcriptomes, organelle, metagenomes and metatranscriptomes, phylogenome analysis. 
 - **Species**: plant, bacterial and fungal.
 - **Sequencing platform** Illumina, PacBio, Oxford Nanopore, PacBioHifi, HiC
 - **Deep learning**: PyTorch, Tensorflow, CatBoost, XGBoost, Sklearn, Keras and other deep learning. 
 - **DevOPs**: Git,Docker,Kubernetes and Terraform for container orchestration.
 - **Generative AI and Graphs.**
 - You can read the curriculum vitae here: [Curriculum Vitae](https://drive.google.com/file/d/10M2skWJsLLx_-Hze7rHxH-bTT-2VJ_if/view?usp=sharing), [Scopus](https://www.scopus.com/authid/detail.uri?authorId=36633064300), [ORCID](https://orcid.org/0000-0002-4157-9405), and [Web of Science](https://www.webofscience.com/wos/author/record/C-5940-2014)
 - You can read the previous employers references and what they say about me [Previous Employers](https://drive.google.com/file/d/1KgQLycgD1S9ztJpt9McQ2yH5lZIUJAgb/view?usp=sharing). 
 - Open to collaboration on Bioinformatics, Deep Learning, Generative AI, and DevOPs.

#### Soft Skills 
- I am very pro-active in making communication and upskilling on emerging trends.
- Proficient in developing new approaches for better integration and minimal interactions, ease of access approaches.
- Profile development by reading about the latest trends in code, personal and professional development.
- Prefer office space work but also open to remote work 1-2 days.
- Efficient in time and project requirements, approaches and solving limitations.
- Voracious reader of language model implementations [language-models](https://paperswithcode.com/)

#### Programming Languages 
- **Bioinformatics, Metabolic Modelling :**  Python, R, Ruby, Awk, Shell, Scala, Nushell, Nextflow, Snakemake, MATLAB
- **Deep Learning:**  Python, R, Scala, learning Julia, learning graphsAPI
- **DevOPs:** Shell, NuShell, Awk, Ruby, Python
- **Web:** Rails, Streamlit, Shiny, Django, BootStrap, Julia, HTML, Javascript
- **Documentation:** Markdown, ReadTheDocs, MkDocs, Jekyll, GitBook
- **Web Engine:** jinja and jinja2, jq, yq

#### Continious Development and Deployment 
- **Git management**: Proficient in Github, Githubcli, Gitlab, GitKraken and Tower for all continous development.
- **Job Scheduler**: Skilled in both pbs and slurm. Established computing cluster and blades.
- **Heroku Development**: Application deployment using [Heroku](https://devcenter.heroku.com/)
- **Docker**: [Docker applications](https://hub.docker.com/u/gauravcodepro)
  
#### [Streamlit and Chainlit Applications](https://streamlit.io/) and [Streamlit Community](https://discuss.streamlit.io/) 
- **streamlit-pacbiohifi**: web application for pacbiohifi sequencing length [streamlit-pacbiohifi](https://pacbiohifi.streamlit.app/) 
- **streamlit-Univeristat-Potsdam**: web application for Universitat Potsdam, [slurm-applicator](https://sup-application.streamlit.app/)

#### [Ruby gems](https://rubygems.org/profiles/gauravcodepro) and [Ruby Community](https://www.ruby-forum.com/) and [Crystal Community](https://forum.crystal-lang.org/): released and development version 
- **devopsutils**: a devops system configuration gem: [devops_system_profiler](https://github.com/gauravcodepro/devops-system)
- **ruby_template_creator**: a ruby templating gem creator: [ruby template creator](https://github.com/gauravcodepro/ruby_gem_creator)
- **fasta_tokenizer**: a fasta tokenizer generator and motif indexer: [fasta_tokenizer](https://github.com/gauravcodepro/pacbiohifi-motif-scanner)
- **panacheextract**: a ruby gem for dealing with snps specific information for panache [panacheextract](https://rubygems.org/gems/panacheextract)
- **fluxmodels**: implementation of all flux models in ruby [fluxmodels](https://github.com/gauravcodepro/flux-models-ruby)
- **protalign**: a pre-release protein alignment gem for the genome annotation [protein-alignment-gem](https://github.com/gauravcodepro/proteinalignment-annotation-gem) and [proteinmultialign](https://github.com/gauravcodepro/protein-multialign-gem)
- **pacbiohifi-analyzer**: analyzing pacbio hifi data from reads to graphs [pacbiohifi-analyzer](https://github.com/gauravcodepro/pacbiohifi-analyzer)

#### [Python Packages](https://pypi.org/user/gauravcodepro/) and [Python Community](https://www.python.org/community/): building development release version 
- **tairaccession**: python package for interacting with tair: [tairaccession](https://github.com/gauravcodepro/tairaccession)
- **graphanalyzer**: python package for analyzing the alignments from the graph alignment tools: [graphanalyzer](https://github.com/gauravcodepro/graphanalyzer)
- **rnaprocessor**: python package for analyzing the sequence for tag based machine learning: [rnaprocessor](https://github.com/gauravcodepro/rnaprocessor)
- **protein-annotator**: python package for analyzing the genome-protein alignments [protein-annotator](https://github.com/gauravcodepro/protein-annotator)
- **ontologyanalyzer**: python for analyzing the ontologies using the semantic web [ontologyanalyzer](https://github.com/gauravcodeproontologyanalyzer)

#### [Julia Packages](https://juliahub.com/): building development release version
- **panachegraph** genome graph and nearest neighbour approach in julia-lang to the pangenome graphs. [panachegraph](https://github.com/gauravcodepro/panachegraph)

#### [R package](https://cran.r-project.org/web/packages/) and [Bioconductor](https://bioconductor.org/) and [Posit community](https://forum.posit.co/): building development release version 
- **evoseq** genome annotation visualization, phylogeny and blocks estimation from protein to genome alignments [evoseq](https://github.com/gauravcodepro/evoseq-genome-informatics), subfunction [genomevisual](https://github.com/gauravcodepro/genome-annotation-visualizer) 
