<p align="left"><img src="/github-metrics.svg" alt="Metrics" width="500"></p>

- I work as Academic Staff Member at [Institute for Biochemistry and Biology](https://www.uni-potsdam.de/en/ibb-bioinformatik/members/gaurav-sablok) at [Universtat Potsdam(UP)](https://www.uni-potsdam.de/de/) :de: 
- I work at the interface of both academics as well as developing applications. 
- [Covering Letter](https://drive.google.com/file/d/1uGG3JZ9TlD1VytRy9DWMElfo0_n-xUo3/view?usp=sharing) [Curriculum Vitae](https://drive.google.com/file/d/1P9BUeprHiRCU1u31O8zIPil47m1Te-AP/view?usp=sharing), [Scopus](https://www.scopus.com/authid/detail.uri?authorId=36633064300), [ORCID](https://orcid.org/0000-0002-4157-9405), and [Web of Science](https://www.webofscience.com/wos/author/record/C-5940-2014)
- [Employers references](https://drive.google.com/file/d/18rUaXpZDNzblcL3txNppogiBhNvIjZVv/view?usp=sharing).
- Contact me at [gaurav.sablok@uni-potsdam.de](mailto:gaurav.sablok@uni-potsdam.de) / [gauravcodepro@gmail.com](mailto:gauravcodepro@gmail.com)
- Communicative, approachable, motivated, task and solution oriented, time responsive, and initiative taker.
- All code and ideas concieved by me unless specified.
- I learn quickly what is required and i am an applicative person.
- Proficient in developing new approaches for code development.
- I read, code and plan well ahead as to have a planned approach of problem solving.
- Prefer onsite work but also open to work in a hybrid environment.
- Active time and project based approaches and solving limitations.
- Voracious reader of language model implementations [language-models](https://paperswithcode.com/) 
- Open to collaboration on Bioinformatics, Deep Learning, Language Deep models, and DevOPs.
- Very collaborative and open to supporting those who support me.
- Not attached to any specific programming language and select according to the need of the work.
- Believe in not wasting time at any aspect [Please read this to save your time](https://github.com/gauravcodepro/gauravcodepro/blob/main/approach.md)
- I dont code C and C++ anymore and follow time based programming to increase my speed and efficiency.

**Professional Aptitude**
- I read scientific aptitude to increase my work potential for continous development:
- I think very precisely before speaking as what has spoken cant be fetched back.
- Pro-active in making effective communication as communication leads to result oriented approaches.
- Bring your skills to the work and show them as that will raise your value.
- I raise my value and respect those who respect my skills and value.
- If there is no outcome or visbility means that there is no existence and baseless and non-existence words have no value. 
- There is no limit but if you put your efforts right, everyday you see a new skill added.
- Effectively work with those, who talk to you and not other way around. Indirect approaches never generate results. 

**Bioinformatics**
- Plant, bacterial and fungal genomes, transcriptomes, organelle, metagenomics and phylogenomics.
- Illumina, PacBio, Oxford Nanopore, PacBioHifi, HiC
- Data Analysis Programming specifics: Python, R, Ruby, MATLAB, Julia
- Web Development Programming specifics: Python, Javascript, Ruby on Rails, Django, HTML and CSS.
- Server Management and Command Line Data Management: Awk, Shell, Nextflow
- Database Management Specifics: MySQL, MongoDB.
- Documentation Programming specifics: Markdown, ReadTheDocs, MkDocs, Jekyll, Github Pages
  
**Deep Learning, RAG, Language Models and Graphs.**
- PyTorch, Tensorflow, CatBoost, XGBoost, Sklearn, Keras, LightGBM, fastai and others. 
- Data Analysis Programming specifics: Python, R, MATLAB, Julia
- Web Development Programming specifics: Python, Javascript, Ruby on Rails, Django, HTML and CSS, Julia
- Database Management Specifics: MySQL, MongoDB.
- Documentation Programming specifics: Markdown, ReadTheDocs, MkDocs, Jekyll, Github Pages
  
**DevOPs**
- Git,GitKraken,Docker,Kubernetes,Terraform for code management and container orchestration.
- Programming specifics: Shell, NuShell, Awk, Python
- PBS and SLURM as job scheduler
- [Heroku](https://devcenter.heroku.com/) and [Docker](https://hub.docker.com/u/gauravcodepro) for application deployment
- Database Management Specifics: MySQL, MongoDB.
- Documentation Programming specifics: Markdown, ReadTheDocs, MkDocs, Jekyll, Github Pages

**Web Develpoment**
- Programming specifics: Ruby on Rails, Streamlit, Python Shiny and ShinyExpress, Django, BootStrap, Julia, HTML, Javascript
- Format: jinja and jinja2, jq, yq
- Database Management Specifics: MySQL, MongoDB.
- Documentation Programming specifics: Markdown, ReadTheDocs, MkDocs, Jekyll, Github Pages

**Streamlit**: [Streamlit Applications](https://streamlit.io/) and [Streamlit Community](https://discuss.streamlit.io/) 
| Name | Description | Application |
|----------|-------------|------|
| Pacbiohifi | pacbiohifi sequencing information | [streamlit-pacbiohifi](https://pacbiohifi.streamlit.app/) |
| Univeristat-Potsdam | web application for Universitat Potsdam | [slurm-applicator](https://sup-application.streamlit.app/) |
| Ontology-graphs | analyzing arabidopsis ontologies and plant obo | [ontology-graphs](https://app-arabidopsis-ontology-graphs.streamlit.app/) |
| Metabolic-json | analyzing BIGG metabolic models | [metabolic-app](https://metabolic-json-modelling.streamlit.app/) |
| Metabolic-analyzer | API to BIGG database models | [metabolic-analyzer](https://github.com/gauravcodepro/streamlit-BIGG-metabolic-analyzer) |

**Ruby Gems**: [Ruby gems](https://rubygems.org/profiles/gauravcodepro), [Ruby app](https://www.ruby-forum.com/) and [Ruby Community](https://www.ruby-forum.com/): released and development version 
| Name | Description | Gem Release |
|----------|-------------|------|
| Devopsutils | devops system configuration | [devops_system_profiler](https://github.com/gauravcodepro/devops-system) |
| Ruby-template | ruby templating  | [ruby template creator](https://github.com/gauravcodepro/ruby_gem_creator) |
| Fasta-tokenizer | fasta tokenizer and motif indexer | [fasta_tokenizer](https://github.com/gauravcodepro/pacbiohifi-motif-scanner) |
| Panacheextract | snp extractor | [panacheextract](https://rubygems.org/gems/panacheextract) |
| Fluxmodels | flux models | [fluxmodels](https://github.com/gauravcodepro/flux-models-ruby) |
| Protalign | protein alignment analyzer | [protein-alignment-gem](https://github.com/gauravcodepro/proteinalignment-annotation-gem) and [proteinmultialign](https://github.com/gauravcodepro/protein-multialign-gem) |
| Pacbiohifi-analyzer | pacbio hifi data from reads to graphs | [pacbiohifi-analyzer](https://github.com/gauravcodepro/pacbiohifi-analyzer) |

**Python Package**: [Python Packages](https://pypi.org/user/gauravcodepro/) and [Python Community](https://www.python.org/community/): building development release version 
| Name | Description | Python Package Repo |
|----------|-------------|------|
| Tairaccession |  python package for interacting with tair | [tairaccession](https://github.com/gauravcodepro/tairaccession) |
| Graphanalyzer | python package for graph alignment tools | [graphanalyzer](https://github.com/gauravcodepro/graphanalyzer) |
| Rnaprocessor | python package for sequence based machine learning | [rnaprocessor](https://github.com/gauravcodepro/rnaprocessor) |
| Protein-annotator | python package for genome-protein alignments | [protein-annotator](https://github.com/gauravcodepro/protein-annotator) |
| Ontologyanalyzer | python package ontologies using the semantic web | [ontologyanalyzer](https://github.com/gauravcodeproontologyanalyzer) |

**R package**: [R package](https://cran.r-project.org/web/packages/) and [Bioconductor](https://bioconductor.org/) and [Posit community](https://forum.posit.co/): building development release version 
| Name | Description | R Package Repo |
|----------|-------------|------|
| Evoseq |  genome annotation visualization, phylogeny | [evoseq](https://github.com/gauravcodepro/evoseq-genome-informatics), subfunction [genomevisual](https://github.com/gauravcodepro/genome-annotation-visualizer) |
