
**About me**

- **Open to collaboration and position on Bioinformatics, Deep Learning, Language Deep models,and DevOPs.**
- **Candidate word: If i have applied for your open position then i am surely interested**
- **Read this:** [Immediate joining and relocation](https://github.com/gauravcodepro/gauravcodepro/blob/main/work.md), [Past collaborator and employers](https://github.com/gauravcodepro/gauravcodepro/blob/main/approach.md), [Mentality Note](https://github.com/gauravcodepro/gauravcodepro/blob/main/mentality.md), [Professionalism](https://github.com/gauravcodepro/gauravcodepro/blob/main/personality.md)
- Application package: [Covering Letter](https://github.com/gauravcodepro/gauravcodepro/blob/main/covering_letter.pdf), [Curriculum Vitae](https://github.com/gauravcodepro/gauravcodepro/blob/main/Curriculum_Vitae_Gaurav_Sablok_2024_7_10.pdf), [Scopus](https://www.scopus.com/authid/detail.uri?authorId=36633064300), [ORCID](https://orcid.org/0000-0002-4157-9405), [Web of Science](https://www.webofscience.com/wos/author/record/C-5940-2014). [Employers references](https://github.com/gauravcodepro/gauravcodepro/blob/main/references.pdf).
- Approachable at [gauravcodepro@gmail.com](mailto:gauravcodepro@gmail.com)
- All code and ideas concieved and written by me unless specified.
- Voracious reader of language model implementations [language-models](https://paperswithcode.com/)
- **Continous learning, and latest feature addition in the priority languages and areas of interest**
- **Prior to 2022, all the code I wrote was internally hosted by the research institutions/universities.** 

[**Work Attitude**](https://github.com/gauravcodepro/gauravcodepro/blob/main/mentality.md) and [**Professionalism**](https://github.com/gauravcodepro/gauravcodepro/blob/main/personality.md)
- **Decisions and approaches/attempts not made in time have no value and importance.**
- I read scientific aptitude to increase my work potential for continous development.
- Communicative, approachable, motivated, task and solution oriented, time responsive, and initiative taker.
- Pro-active in making effective communication as communication leads to result oriented approaches.
- Active in solving limitations and applicative person, proficient in developing new approaches for code development.
- Plan and code well ahead as to have a planned approach of problem solving.
- I think very precisely before speaking as what has spoken cant be fetched back.
- I belive in effectively working with those, who talk to me and not other way around.
- I belive in not taking and appreciating indirect approaches as it never generate results.
- Extremely collaborative and open to supporting those who support me.
- I dont work with the past and this keeps me result oriented and give importance to the people i am working with.
- I pick the programming langauge to get the work done and can drop anytime and pick another.

**Areas of Interest**
 
- **Bioinformatics,Deep Learning,RAG,Language Models and Graphs.**
  - Priority Language: Python, R, Ruby, Julia, Javascript, Shell, Awk, Nextflow
  - Machine/Deep Learning:PyTorch, Tensorflow, CatBoost, XGBoost, Sklearn, Keras, LightGBM, Fastai.
  - Package Development: Python, Julia, Javascript, Ruby.
  - API Development using FASTAPI/Javascript
  - Documentation: Markdown, ReadTheDocs, MkDocs, Jekyll, Github Pages, Quatro

- **DevOPs**
  - Priority Language: Shell, NuShell, Awk, Python
  - Code Management: Git, GitKraken.
  - Application deployment: [Docker](https://hub.docker.com/u/gauravcodepro)
  - Orchestration:Terraform, Kubernetes.
  - Database Management: SQlite3, MySQL, MongoDB.
  - PBS and SLURM as job scheduler

- **Web Develpoment**
  - Priority Language: Ruby on Rails, Streamlit, Python Shiny/ShinyExpress, Django, BootStrap, Julia, HTML, Javascript
  - JSON Format: jinja and jinja2, jq, yq

- **Application Development**
  - Learning Electron, Slint, Tauri, PyQT, PyGTK 

**Streamlit**: [Streamlit Applications](https://streamlit.io/) and [Streamlit Community](https://discuss.streamlit.io/)
 - [Metabolic-analyzer:API to BIGG database models](https://github.com/gauravcodepro/streamlit-BIGG-metabolic-analyzer-API)
 - [Metabolic-json: analyzing BIGG metabolic models](https://github.com/gauravcodepro/streamlit-metabolic-json-modelling)
 - [Ontology-graphs: analyzing arabidopsis ontologies and plant obo](https://github.com/gauravcodepro/streamlit-arabidopsis-ontology-graphs)
 - [Pacbiohifi: pacbiohifi sequencing information](https://github.com/gauravcodepro/streamlit-pacbiohifi)
 - [DevOPS Application Slurm Generator](https://github.com/gauravcodepro/streamlit-up-application)

**Ruby Gems**: [Ruby gems](https://rubygems.org/profiles/gauravcodepro), [Ruby app](https://www.ruby-forum.com/) and [Ruby Community](https://www.ruby-forum.com/): released and development version 
 - [Devopsutils: devops system configuration](https://github.com/gauravcodepro/devops-system)
 - [Ruby-template: ruby templating](https://github.com/gauravcodepro/ruby_gem_creator)
 - [Fasta-tokenizer: fasta tokenizer and motif indexer](https://github.com/gauravcodepro/pacbiohifi-motif-scanner)
 - [Panacheextract: snp extractor](https://rubygems.org/gems/panacheextract) 
 - [Protalign: protein alignment analyzer](https://github.com/gauravcodepro/proteinalignment-annotation-gem) and [proteinmultialign](https://github.com/gauravcodepro/protein-multialign-gem) 
 - [Pacbiohifi-analyzer: pacbio hifi data from reads to graphs](https://github.com/gauravcodepro/pacbiohifi-analyzer) 

**Python Package**: [Python Packages](https://pypi.org/user/gauravcodepro/) and [Python Community](https://www.python.org/community/): building development release version 
 - [Tairaccession: python package for interacting with tair](https://github.com/gauravcodepro/tairaccession) 
 - [Graphanalyzer: python package for graph alignment tools](https://github.com/gauravcodepro/graphanalyzer)
 - [RNAprocessor: python package for sequence based machine learning](https://github.com/gauravcodepro/rnaprocessor)
 - [ProteinAnnotator: python package for genome-protein alignments](https://github.com/gauravcodepro/protein-annotator)
 - [Ontologyanalyzer: python package ontologies using the semantic web](https://github.com/gauravcodeproontologyanalyzer) 
