
- **Open to collaboration and positons on Bioinformatics, Deep Learning, Language Deep models,and DevOPs.**
- [Please read this: Immediate joining and relocation for the selection position](https://github.com/gauravcodepro/gauravcodepro/blob/main/work.md)
- [Covering Letter](https://github.com/gauravcodepro/gauravcodepro/blob/main/covering_letter.pdf), [Curriculum Vitae](https://github.com/gauravcodepro/gauravcodepro/blob/main/Curriculum_Vitae_Gaurav_Sablok_2024_7_10.pdf), [Scopus](https://www.scopus.com/authid/detail.uri?authorId=36633064300), [ORCID](https://orcid.org/0000-0002-4157-9405), [Web of Science](https://www.webofscience.com/wos/author/record/C-5940-2014). [Employers references](https://github.com/gauravcodepro/gauravcodepro/blob/main/references.pdf).
- I can be contacted at [gauravcodepro@gmail.com](mailto:gauravcodepro@gmail.com)
- All code and ideas concieved by me unless specified.
- Voracious reader of language model implementations [language-models](https://paperswithcode.com/) 
- Believe in not wasting time at any aspect [Please read this to save your time](https://github.com/gauravcodepro/gauravcodepro/blob/main/approach.md)
- I dont code C and C++ anymore and follow time based programming to increase my speed and efficiency.
- If you a previous collaborator and i havent seen you ever and talked to you or not been in contact with you in the last years, please dont write/approach me. I am not interested.

**Working Attitude**
- I read scientific aptitude to increase my work potential for continous development.
- Communicative, approachable, motivated, task and solution oriented, time responsive, and initiative taker.
- Pro-active in making effective communication as communication leads to result oriented approaches.
- Active in solving limitations and applicative person, proficient in developing new approaches for code development.
- Plan and code well ahead as to have a planned approach of problem solving.
- I think very precisely before speaking as what has spoken cant be fetched back.
- I raise my value and respect those who respect my skills and value.
- I belive in effectively working with those, who talk to me and not other way around.
- I belive in not taking and appreciating indirect approaches as it never generate results.
- Extremely collaborative and open to supporting those who support me.
- I never delay in making a contact, as delay means not interested and interested means you dont delay.
- Following language specific code repositories.
- Not interested in themes, fontsizes, theme colours or any type of such associations. 

**Bioinformatics,Deep Learning,RAG,Language Models and Graphs.**
- Language: Python, R, Ruby, Julia
- Machine/Deep Learning:PyTorch, Tensorflow, CatBoost, XGBoost, Sklearn, Keras, LightGBM, Fastai.
- Web Development: Python, Javascript, Julia, Ruby on Rails, Django, HTML and CSS.
- Server and Data Management: Awk, Shell, Nextflow
- Database Management: SQlite3, MySQL, MongoDB.
- Documentation: Markdown, ReadTheDocs, MkDocs, Jekyll, Github Pages, Quatro
- Package Development: Python, Julia, Javascript, Ruby.
- API Development using FASTAPI/Javascript
  
**DevOPs**
- Code Management: Git,GitKraken,Docker,Kubernetes,Terraform.
- Language: Shell, NuShell, Awk, Python
- Database: SQlite3, MySQL, MongoDB.
- Documentation: Markdown, ReadTheDocs, MkDocs, Jekyll, Github Pages, Quatro
- PBS and SLURM as job scheduler
- [Docker](https://hub.docker.com/u/gauravcodepro) for application deployment

**Web Develpoment**
- Language: Ruby on Rails, Streamlit, Python Shiny/ShinyExpress, Django, BootStrap, Julia, HTML, Javascript
- JSON Format: jinja and jinja2, jq, yq

**Streamlit**: [Streamlit Applications](https://streamlit.io/) and [Streamlit Community](https://discuss.streamlit.io/) 
| Name | Application |
|----------|------|
| Pacbiohifi: pacbiohifi sequencing information | [Pacbiohifi](https://pacbiohifi.streamlit.app/) |
| Univeristat-Potsdam: web application for Universitat Potsdam | [slurm-applicator](https://sup-application.streamlit.app/) |
| Ontology-graphs: analyzing arabidopsis ontologies and plant obo | [ontology-graphs](https://app-arabidopsis-ontology-graphs.streamlit.app/) |
| Metabolic-json: analyzing BIGG metabolic models | [metabolic-app](https://metabolic-json-modelling.streamlit.app/) |
| Metabolic-analyzer:API to BIGG database models | [metabolic-analyzer](https://github.com/gauravcodepro/streamlit-BIGG-metabolic-analyzer-API) |

**Ruby Gems**: [Ruby gems](https://rubygems.org/profiles/gauravcodepro), [Ruby app](https://www.ruby-forum.com/) and [Ruby Community](https://www.ruby-forum.com/): released and development version 
| Name | Gem Release |
|----------|------|
| Devopsutils: devops system configuration | [devops_system_profiler](https://github.com/gauravcodepro/devops-system) |
| Ruby-template: ruby templating  | [ruby template creator](https://github.com/gauravcodepro/ruby_gem_creator) |
| Fasta-tokenizer: fasta tokenizer and motif indexer | [fasta_tokenizer](https://github.com/gauravcodepro/pacbiohifi-motif-scanner) |
| Panacheextract: snp extractor | [panacheextract](https://rubygems.org/gems/panacheextract) |
| Protalign: protein alignment analyzer | [protein-alignment-gem](https://github.com/gauravcodepro/proteinalignment-annotation-gem) and [proteinmultialign](https://github.com/gauravcodepro/protein-multialign-gem) |
| Pacbiohifi-analyzer: pacbio hifi data from reads to graphs | [pacbiohifi-analyzer](https://github.com/gauravcodepro/pacbiohifi-analyzer) |

**Python Package**: [Python Packages](https://pypi.org/user/gauravcodepro/) and [Python Community](https://www.python.org/community/): building development release version 
| Name | Python Package Repo |
|----------|------|
| Tairaccession: python package for interacting with tair | [tairaccession](https://github.com/gauravcodepro/tairaccession) |
| Graphanalyzer: python package for graph alignment tools | [graphanalyzer](https://github.com/gauravcodepro/graphanalyzer) |
| RNAprocessor: python package for sequence based machine learning | [rnaprocessor](https://github.com/gauravcodepro/rnaprocessor) |
| ProteinAnnotator: python package for genome-protein alignments | [protein-annotator](https://github.com/gauravcodepro/protein-annotator) |
| Ontologyanalyzer: python package ontologies using the semantic web | [ontologyanalyzer](https://github.com/gauravcodeproontologyanalyzer) |
