Highly motivated and skilled personnel actively working across Bioinformatics, Data Science and Analytics, Machine and Deep Learning. My areas of focus are Bioinformatics, DataScience and Analytics, Machine and Deep Learning. I use Python, R, Shell, Nu, Ruby, Awk, Crystal. I am coding both in R and Python shiny application and also use rust based nushell programming for many aspects from data exploration to datascience and bioinformatics. I work with large scale sequencing (transcriptomics, genome sequencing metagenomics and single cell genomics(plants specific) and also spatial transcriptomics) involving the assembly of the plant genomes from high order to lower order, chloroplast and mitochondrial genomes, bacterial genomes, metagenomics and fungal genomes coming from Illumina Seq, MiSeq, NextSeq, PacBio, Oxford Nanopore. Having an extensive experience in molecular biology, bioinformatics, comparative genomics and biological mining of large-scale data-sets and databases in plant, bacterial and fungal species to unravel the biological basis. 
Skilled in both pbs and slurm based high performance computing clusters, devops management and computing clusters and uses git, docker and kubernetes for container orchestration. 

Happy New Year 2024 and with this i completed Hashicorp programming language on the eve of the Happy New Year 2023. 

If you would like to connect then you can connect with me below

![Your Repository's Stats](https://github-readme-stats.vercel.app/api?username=sablokgaurav&show_icons=true)

Github:https://github.com/sablokgaurav \
ORCID: https://orcid.org/0000-0002-4157-9405 \
WOS: https://www.webofscience.com/wos/author/record/C-5940-2014 \
RubyGems Published: https://rubygems.org/profiles/sablokgaurav \
Python Packages Published : https://pypi.org/user/sablokgaurav/ \
Crystal Profile: https://forum.crystal-lang.org/u/sablokgaurav/ \
devProfile: https://app.daily.dev/sablokgaurav \
assumptions lead to failures and efforts lead to success. 


