#### About me

:point_right: I'm based in Germany as Academic Staff Member at Bioinformatics, Institute for Biochemistry and Biology at [Universtat Potsdam(UP)](https://www.uni-potsdam.de/de/) and [Group](https://www.uni-potsdam.de/en/ibb-bioinformatik/members/gaurav-sablok) :de: \
:point_right:  You can contact me at [gaurav.sablok@uni-potsdam.de](mailto:gaurav.sablok@uni-potsdam.de) and [sablokg@gmail.com](mailto:sablokg@gmail.com) \
:point_right: Highly communicative, motivated, task oriented, feature responsive, time oriented, approachable, solution seeker and initiative taker focussed professional person. \
:point_right: **Bioinformatics |  Deep Learning | DevOps | Generative AI | Graphs** \
:point_right: **Bioinformatics**: genomes, transcriptomes, organelle, metagenomes and metatranscriptomes, evolutionary and phylogenome analysis. \
:point_right: **Species**: plant, bacterial and fungal. \
:point_right: **Sequencing platform** Illumina, PacBio, Oxford Nanopore. \
:point_right: **Deep learning**: PyTorch, Tensorflow, CatBoost, XGBoost, Sklearn, Keras and other machine and deep learning \
:point_right: **DevOPs**: Git,Docker,Kubernetes and Terraform for container orchestration. \
:point_right: **Job Scheduler**: Skilled in both pbs and slurm. Established computing cluster and blades. \
:point_right: **Git management**: Github commandline and GitKraken inlcuding stagging, rebasing, stashing, commiting \
:point_right: **Generative AI and Graphs.** \
:point_right: You can read the curriculum vitae here: [curriculum vitae](https://drive.google.com/file/d/1cmWwiBgFG40HnZxjxb913r-jjjuDAGFk/view?usp=sharing) and [citations](https://scholar.google.com/citations?hl=de&user=XaA2hbUAAAAJ&view_op=list_works&sortby=pubdate) and [web of science](https://www.webofscience.com/wos/author/record/C-5940-2014)  \
:point_right: You can read the employers references and what they say about me [employers](https://drive.google.com/file/d/1nRod167faP1GhjIUS1UCpd7iMBTAIUDf/view?usp=sharing). \
:point_right: Hosting the completed packages at [codearn](https://github.com/codeearn) \
:point_right: Open to collaboration on Bioinformatics, Deep Learning, Generative AI, and DevOPs.
#### Soft Skills
:point_right: I am very pro-active in making communication and upskilling on emerging trends. \
:point_right: Proficient in developing new approaches for better integration and minimal interactions, ease of access approaches. \
:point_right: Profile development by reading about the latest trends in code, personal and professional development. \
:point_right: Prefer office space work but also open to remote work 1-2 days. \
:point_right: Efficient in time and project requirements, approaches and solving limitations.

#### Programming Languages
![HTML5](https://img.shields.io/badge/html5-%23E34F26.svg?style=plastic&logo=html5&logoColor=white) ![CSS3](https://img.shields.io/badge/css3-%231572B6.svg?style=plastic&logo=css3&logoColor=white) 	![Julia](https://img.shields.io/badge/-Julia-9558B2?style=plastic&logo=julia&logoColor=white) ![Python](https://img.shields.io/badge/python-3670A0?style=plastic&logo=python&logoColor=ffdd54) ![R](https://img.shields.io/badge/r-%23276DC3.svg?style=plastic&logo=r&logoColor=white) ![Shell Script](https://img.shields.io/badge/shell_script-%23121011.svg?style=plastic&logo=gnu-bash&logoColor=white) ![Ruby](https://img.shields.io/badge/ruby-%23CC342D.svg?style=plastic&logo=ruby&logoColor=white) ![Markdown](https://img.shields.io/badge/markdown-%23000000.svg?style=plastic&logo=markdown&logoColor=white) ![Anaconda](https://img.shields.io/badge/Anaconda-%2344A833.svg?style=plastic&logo=anaconda&logoColor=white) ![Apollo-GraphQL](https://img.shields.io/badge/-ApolloGraphQL-311C87?style=plastic&logo=apollo-graphql) ![GraphQL](https://img.shields.io/badge/-GraphQL-E10098?style=plastic&logo=graphql&logoColor=white) ![Django](https://img.shields.io/badge/django-%23092E20.svg?style=plastic&logo=django&logoColor=white) ![DjangoREST](https://img.shields.io/badge/DJANGO-REST-ff1709?style=plastic&logo=django&logoColor=white&color=ff1709&labelColor=gray) ![FastAPI](https://img.shields.io/badge/FastAPI-005571?style=plastic&logo=fastapi) ![Rails](https://img.shields.io/badge/rails-%23CC0000.svg?style=plastic&logo=ruby-on-rails&logoColor=white) ![TailwindCSS](https://img.shields.io/badge/tailwindcss-%2338B2AC.svg?style=plastic&logo=tailwind-css&logoColor=white) ![Jenkins](https://img.shields.io/badge/jenkins-%232C5263.svg?style=plastic&logo=jenkins&logoColor=white) ![Keras](https://img.shields.io/badge/Keras-%23D00000.svg?style=plastic&logo=Keras&logoColor=white) ![NumPy](https://img.shields.io/badge/numpy-%23013243.svg?style=plastic&logo=numpy&logoColor=white) ![Pandas](https://img.shields.io/badge/pandas-%23150458.svg?style=plastic&logo=pandas&logoColor=white) ![Plotly](https://img.shields.io/badge/Plotly-%233F4F75.svg?style=plastic&logo=plotly&logoColor=white) ![PyTorch](https://img.shields.io/badge/PyTorch-%23EE4C2C.svg?style=plastic&logo=PyTorch&logoColor=white) ![scikit-learn](https://img.shields.io/badge/scikit--learn-%23F7931E.svg?style=plastic&logo=scikit-learn&logoColor=white) ![SciPy](https://img.shields.io/badge/SciPy-%230C55A5.svg?style=plastic&logo=scipy&logoColor=%white) ![TensorFlow](https://img.shields.io/badge/TensorFlow-%23FF6F00.svg?style=plastic&logo=TensorFlow&logoColor=white) ![Ansible](https://img.shields.io/badge/ansible-%231A1918.svg?style=plastic&logo=ansible&logoColor=white) ![Kubernetes](https://img.shields.io/badge/kubernetes-%23326ce5.svg?style=plastic&logo=kubernetes&logoColor=white) ![Docker](https://img.shields.io/badge/docker-%230db7ed.svg?style=plastic&logo=docker&logoColor=white) ![Terraform](https://img.shields.io/badge/terraform-%235835CC.svg?style=plastic&logo=terraform&logoColor=white) ![Gitkraken](https://img.shields.io/badge/GitKraken-179287?style=plastic&logo=GitKraken&logoColor=white)

:point_right:  **Currently program:** \
:point_right: “Progress thrives on the exchange of knowledge.” (Albert Einstein) \
:point_right: **Bioinformatics:**  Python, R, Ruby, Awk, Shell, Nushell \
:point_right: **Deep Learning:**  Python, R, learning Julia, graphsAPI. \
:point_right: **DevOPs:** Shell, NuShell, Awk, Ruby, HashiCorp, Python \
:point_right: **Web Application:** Streamlit, Django, HTML and Tailwind CSS \
:point_right: **Static documentation:** Markdown, ReadTheDocs, MKdocs, Jekyll and Ruby  \
:point_right: **Templating engine:** jinja and jinja2, jq and yq \
:point_right: **Bioinformatics templating:** Nextflow, Snakemake \
:point_right: **Previously program:** C++ and Scala.

### Operating system and Containerization:
:point_right: Currently using Linux [Linux](https://linuxcommunity.io/u/sablokgaurav/summary) and [plasmakde](https://kde.org/) \
:point_right: Previously using MacBook. \
:point_right: [Discord](https://discord.com/) **sablokgaurav** \
:point_right: [Docker applications](https://hub.docker.com/u/sablokg)

#### [Python Packages](https://pypi.org/user/sablokgaurav/): development and release version 
:point_right: **tairaccession**: python package for interacting with tair: [tairaccession](https://github.com/sablokgaurav/tairaccession) \
:point_right: **graphanalyzer**: python package for analyzing the alignments from the graph alignment tools: [graphanalyzer](https://github.com/sablokgaurav/graphanalyzer) \
:point_right: **rnaprocessor**: python package for analyzing the sequence for tag based machine learning: [rnaprocessor](https://github.com/sablokgaurav/rnaprocessor) \
:point_right: **protein-annotator**: python package for analyzing the genome-protein alignments [protein-annotator](https://github.com/sablokgaurav/protein-annotator) \
:point_right: **ontologyanalyzer**: python for analyzing the ontologies using the semantic web [ontologyanalyzer](https://github.com/sablokgaurav/ontologyanalyzer)
#### [Ruby gems](https://rubygems.org/profiles/sablokgaurav): released and development version
:point_right: **devops_system_profiler**: a devops system configuration gem: [devops_system_profiler](https://github.com/sablokgaurav/devops-system) \
:point_right: **ruby_template_creator**: a ruby templating gem creator: [ruby template creator](https://github.com/sablokgaurav/ruby_gem_creator) \
:point_right: **fasta_tokenizer**: a fasta tokenizer generator and motif indexer: [fasta_tokenizer](https://github.com/sablokgaurav/pacbiohifi-motif-scanner) \
:point_right: **panacheextract**: a ruby gem for dealing with snps specific information for panache [panacheextract](https://rubygems.org/gems/panacheextract) \
:point_right: **fluxmodels**: implementation of all flux models in ruby [fluxmodels](https://github.com/sablokgaurav/flux-models-ruby) \
:point_right: **protalign**: a pre-release protein alignment gem for the genome annotation [protein-alignment-gem](https://github.com/sablokgaurav/proteinalignment-annotation-gem) and [proteinmultialign](https://github.com/sablokgaurav/protein-multialign-gem) \
:point_right: **pacbiohifi-analyzer**: analyzing pacbio hifi data from reads to graphs [pacbiohifi-analyzer](https://github.com/sablokgaurav/pacbiohifi-analyzer)

#### [Julia Packages](https://juliahub.com/): building release version
:point_right: **panachegraph** genome graph and nearest neighbour approach in julia-lang to the pangenome graphs. [panachegraph](https://github.com/sablokgaurav/panachegraph)
