- I work as Academic Staff Member at Bioinformatics, Institute for Biochemistry and Biology at [Universtat Potsdam(UP)](https://www.uni-potsdam.de/de/) and [Group](https://www.uni-potsdam.de/en/ibb-bioinformatik/members/gaurav-sablok) :de: 
- I work at the interface of both academics as well as developing applications from client side rendering. 
- You can contact me at [gaurav.sablok@uni-potsdam.de](mailto:gaurav.sablok@uni-potsdam.de)
- If you want to contact me on Gmail, you can use [gauravcodepro@gmail.com](mailto:gauravcodepro@gmail.com)
- Communicative, approachable, motivated, task and solution oriented, time responsive, and initiative taker.
- Pro-active in making communication and upskilling on emerging trends.
- Proficient in developing new approaches for code development and have been coding for more than 14 years.
- Prefer onsite work but also open to work in a hybrid environment.
- Active time and project based approaches and solving limitations.
- Voracious reader of language model implementations [language-models](https://paperswithcode.com/)
- Curriculum vitae here: [Covering Letter](https://drive.google.com/file/d/1uGG3JZ9TlD1VytRy9DWMElfo0_n-xUo3/view?usp=sharing) [Curriculum Vitae](https://drive.google.com/file/d/1P9BUeprHiRCU1u31O8zIPil47m1Te-AP/view?usp=sharing), [Scopus](https://www.scopus.com/authid/detail.uri?authorId=36633064300), [ORCID](https://orcid.org/0000-0002-4157-9405), and [Web of Science](https://www.webofscience.com/wos/author/record/C-5940-2014)
- Employers references [Previous Employers](https://drive.google.com/file/d/18rUaXpZDNzblcL3txNppogiBhNvIjZVv/view?usp=sharing). 
- Open to collaboration on Bioinformatics, Deep Learning, Generative AI, and DevOPs.
- Very collaborative and open to supporting those who support me.
 
**Bioinformatics**
- Plant, bacterial and fungal genomes, transcriptomes, organelle, metagenomics and phylogenomics.
- Illumina, PacBio, Oxford Nanopore, PacBioHifi, HiC
- Data Analysis Programming specifics: Python, R, Ruby, Awk, Shell, Nextflow, Snakemake, MATLAB, Julia
- Web Development Programming specifics: Python, Javascript, Ruby on Rails, Django, HTML and CSS. 
  
**Deep Learning, RAG, Language Models and Graphs.**
- PyTorch, Tensorflow, CatBoost, XGBoost, Sklearn, Keras, LightGBM and others. 
-  Data Analysis Programming specifics: Python, R, MATLAB, learning Julia and graphsAPI
- Web Development Programming specifics: Python, Javascript, Ruby on Rails, Django, HTML and CSS, Julia

**DevOPs**
- Git,GitKraken,Docker,Kubernetes,Terraform for code management and container orchestration.
- Programming specifics: Shell, NuShell, Awk, Python
- PBS and SLURM as job scheduler
- [Heroku](https://devcenter.heroku.com/) and [Docker](https://hub.docker.com/u/gauravcodepro) for application deployment

**Documentation**
- Markdown, ReadTheDocs, MkDocs, Jekyll, Github Pages
  
**Web Develpoment**
- Programming specifics: Ruby on Rails, Streamlit, Python Shiny and ShinyExpress, Django, BootStrap, Julia, HTML, Javascript
- Format: jinja and jinja2, jq, yq

**Streamlit**: [Streamlit Applications](https://streamlit.io/) and [Streamlit Community](https://discuss.streamlit.io/) 
| Name | Description | URL |
|----------|-------------|------|
| streamlit-pacbiohifi | pacbiohifi sequencing information | [streamlit-pacbiohifi](https://pacbiohifi.streamlit.app/) |
| streamlit-Univeristat-Potsdam | web application for Universitat Potsdam | [slurm-applicator](https://sup-application.streamlit.app/) |
| streamlit-ontology-graphs | analyzing arabidopsis ontologies and plant obo | [ontology-graphs](https://app-arabidopsis-ontology-graphs.streamlit.app/) |
| streamlit-metabolic-json | analyzing BIGG metabolic models | [metabolic-app](https://metabolic-json-modelling.streamlit.app/) |
| streamlit-metabolic-analyzer | analyzing complete BIGG database from your desktop, An API to BIGG database | [metabolic-model](https://github.com/gauravcodepro/streamlit-BIGG-metabolic-analyzer) |

**Ruby Gems**: [Ruby gems](https://rubygems.org/profiles/gauravcodepro), [Ruby app](https://www.ruby-forum.com/) and [Ruby Community](https://www.ruby-forum.com/): released and development version 
| Name | Description | URL |
|----------|-------------|------|
| devopsutils | a devops system configuration gem | [devops_system_profiler](https://github.com/gauravcodepro/devops-system) |
| ruby_template_creator | a ruby templating gem creator | [ruby template creator](https://github.com/gauravcodepro/ruby_gem_creator) |
| fasta_tokenizer | a fasta tokenizer generator and motif indexer | [fasta_tokenizer](https://github.com/gauravcodepro/pacbiohifi-motif-scanner) |
| panacheextract | snps specific information for panache | [panacheextract](https://rubygems.org/gems/panacheextract) |
| fluxmodels | implementation of flux models | [fluxmodels](https://github.com/gauravcodepro/flux-models-ruby) |
| protalign | protein alignment gem for the genome annotation | [protein-alignment-gem](https://github.com/gauravcodepro/proteinalignment-annotation-gem) and [proteinmultialign](https://github.com/gauravcodepro/protein-multialign-gem) |
| pacbiohifi-analyzer | analyzing pacbio hifi data from reads to graphs | [pacbiohifi-analyzer](https://github.com/gauravcodepro/pacbiohifi-analyzer) |

**Python Package**: [Python Packages](https://pypi.org/user/gauravcodepro/) and [Python Community](https://www.python.org/community/): building development release version 
| Name | Description | URL |
|----------|-------------|------|
| tairaccession |  python package for interacting with tair | [tairaccession](https://github.com/gauravcodepro/tairaccession) |
| graphanalyzer | python package for graph alignment tools | [graphanalyzer](https://github.com/gauravcodepro/graphanalyzer) |
| rnaprocessor | python package for sequence based machine learning | [rnaprocessor](https://github.com/gauravcodepro/rnaprocessor) |
| protein-annotator | python package for genome-protein alignments | [protein-annotator](https://github.com/gauravcodepro/protein-annotator) |
| ontologyanalyzer | python package ontologies using the semantic web | [ontologyanalyzer](https://github.com/gauravcodeproontologyanalyzer) |

**R package**: [R package](https://cran.r-project.org/web/packages/) and [Bioconductor](https://bioconductor.org/) and [Posit community](https://forum.posit.co/): building development release version 
| Nname | Description | URL |
|----------|-------------|------|
| evoseq |  genome annotation visualization, phylogeny from protein to genome alignments | [evoseq](https://github.com/gauravcodepro/evoseq-genome-informatics), subfunction [genomevisual](https://github.com/gauravcodepro/genome-annotation-visualizer) |
