#### About me

:point_right: I'm based in Germany as Academic Staff Member at Bioinformatics, Institute for Biochemistry and Biology at [Universtat Potsdam(UP)](https://www.uni-potsdam.de/de/) and [Group](https://www.uni-potsdam.de/en/ibb-bioinformatik/members/gaurav-sablok) :de: \
:point_right:  Progress thrives on the exchange of knowledge. (Albert Einstein) \
:point_right:  Fortune favours the prepared mind. (Louis Pasteur) \
:point_right:  You can contact me at [gaurav.sablok@uni-potsdam.de](mailto:gaurav.sablok@uni-potsdam.de)  \
:point_right: If you want to contact me on Gmail, you can use [gauravcodepro@gmail.com](mailto:gauravcodepro@gmail.com) \
:point_right: Highly communicative, motivated, task oriented, feature responsive, time oriented, approachable, solution seeker and initiative taker focussed professional person. \
:point_right: **Bioinformatics | Â Deep Learning | DevOps | Generative AI | Graphs** \
:point_right: **Bioinformatics**: genomes, transcriptomes, organelle, metagenomes and metatranscriptomes, evolutionary and phylogenome analysis. \
:point_right: **Species**: plant, bacterial and fungal. \
:point_right: **Sequencing platform** Illumina, PacBio, Oxford Nanopore, PacBioHifi, HiC \
:point_right: **Deep learning**: PyTorch, Tensorflow, CatBoost, XGBoost, Sklearn, Keras and other deep learning. \
:point_right: **DevOPs**: Git,Docker,Kubernetes and Terraform for container orchestration. \
:point_right: **Generative AI and Graphs.** \
:point_right: You can read the curriculum vitae here: [Curriculum Vitae](https://drive.google.com/file/d/1xhPEQmup5iXFC3Bpv8y5jnn9Dm5Itv5F/view?usp=sharing), [Scopus](https://www.scopus.com/authid/detail.uri?authorId=36633064300), [ORCID](https://orcid.org/0000-0002-4157-9405), and [Web of Science](https://www.webofscience.com/wos/author/record/C-5940-2014)  \
:point_right: You can read the previous employers references and what they say about me [Previous Employers](https://drive.google.com/file/d/1nRod167faP1GhjIUS1UCpd7iMBTAIUDf/view?usp=sharing). \
:point_right: Open to collaboration on Bioinformatics, Deep Learning, Generative AI, and DevOPs.
#### Soft Skills
:point_right: I am very pro-active in making communication and upskilling on emerging trends. \
:point_right: Proficient in developing new approaches for better integration and minimal interactions, ease of access approaches. \
:point_right: Profile development by reading about the latest trends in code, personal and professional development. \
:point_right: Prefer office space work but also open to remote work 1-2 days. \
:point_right: Efficient in time and project requirements, approaches and solving limitations.

#### Programming Languages
:point_right: **Bioinformatics:**  Python, R, Ruby, Awk, Shell, Scala, Nushell \
:point_right: **Deep Learning:**  Python, R, Scala, learning Julia, learning graphsAPI. \
:point_right: **DevOPs:** Shell, NuShell, Awk, Ruby, HashiCorp, Python \
:point_right: **Web Application:** Streamlit, Shiny, Django, Julia, HTML, Javascript \
:point_right: **Static documentation:** Markdown, ReadTheDocs, Jekyll and Ruby  \
:point_right: **Templating engine:** jinja and jinja2, jq, yq \
:point_right: **Bioinformatics templating:** Nextflow, Snakemake.  

#### Operating system, Containerization, Open Software:
:point_right: Currently using [Ubuntu](https://kde.org/)  and [Community](https://linuxcommunity.io/) \
:point_right: Previously used MacBook. \
:point_right: [Docker applications](https://hub.docker.com/u/sablokg) \
:point_right: [Journal of Open Source Software](https://joss.theoj.org/)

#### Continious Development and Deployment
:point_right: **Git management**: Proficient in Github, Githubcli, Gitlab, GitKraken and Tower for all continous development. \
:point_right: **Job Scheduler**: Skilled in both pbs and slurm. Established computing cluster and blades. 

#### [Python Packages](https://pypi.org/user/gauravcodepro/): development and release version 
:point_right: **tairaccession**: python package for interacting with tair: [tairaccession](https://github.com/gauravcodepro/tairaccession) \
:point_right: **graphanalyzer**: python package for analyzing the alignments from the graph alignment tools: [graphanalyzer](https://github.com/gauravcodepro/graphanalyzer) \
:point_right: **rnaprocessor**: python package for analyzing the sequence for tag based machine learning: [rnaprocessor](https://github.com/gauravcodepro/rnaprocessor) \
:point_right: **protein-annotator**: python package for analyzing the genome-protein alignments [protein-annotator](https://github.com/gauravcodepro/protein-annotator) \
:point_right: **ontologyanalyzer**: python for analyzing the ontologies using the semantic web [ontologyanalyzer](https://github.com/gauravcodeproontologyanalyzer)

#### [Ruby gems](https://rubygems.org/profiles/gauravcodepro): released and development version
:point_right: **devops_system_profiler**: a devops system configuration gem: [devops_system_profiler](https://github.com/gauravcodepro/devops-system) \
:point_right: **ruby_template_creator**: a ruby templating gem creator: [ruby template creator](https://github.com/gauravcodepro/ruby_gem_creator) \
:point_right: **fasta_tokenizer**: a fasta tokenizer generator and motif indexer: [fasta_tokenizer](https://github.com/gauravcodepro/pacbiohifi-motif-scanner) \
:point_right: **panacheextract**: a ruby gem for dealing with snps specific information for panache [panacheextract](https://rubygems.org/gems/panacheextract) \
:point_right: **fluxmodels**: implementation of all flux models in ruby [fluxmodels](https://github.com/gauravcodepro/flux-models-ruby) \
:point_right: **protalign**: a pre-release protein alignment gem for the genome annotation [protein-alignment-gem](https://github.com/gauravcodepro/proteinalignment-annotation-gem) and [proteinmultialign](https://github.com/gauravcodepro/protein-multialign-gem) \
:point_right: **pacbiohifi-analyzer**: analyzing pacbio hifi data from reads to graphs [pacbiohifi-analyzer](https://github.com/gauravcodepro/pacbiohifi-analyzer)

#### [Julia Packages](https://juliahub.com/): building release version
:point_right: **panachegraph** genome graph and nearest neighbour approach in julia-lang to the pangenome graphs. [panachegraph](https://github.com/gauravcodepro/panachegraph)

#### [R package](https://cran.r-project.org/web/packages/) and [Bioconductor](https://bioconductor.org/): building release version
:point_right: **evoseq** genome annotation visualization, phylogeny and blocks estimation from protein to genome alignments [evoseq](https://github.com/gauravcodepro/evoseq-genome-informatics), subfunction [genomevisual](https://github.com/gauravcodepro/genome-annotation-visualizer) 

#### [Streamlit and Chainlit Applications](https://streamlit.io/)
:point_right: **streamlit-pacbiohifi**: web application for pacbiohifi sequencing length [streamlit-pacbiohifi](https://pacbiohifi.streamlit.app/) \
:point_right: **streamlit-Univeristat-Potsdam**: web application for Universitat Potsdam, [slurm-applicator](https://sup-application.streamlit.app/)

#### [Heroku Applications](https://www.heroku.com/)
:point_right: **model ontologies**: web application deployed for modelling ontologies on Heroku
